code/_core/obj/item/material/_material.dm

```diff
@@ -50,6 +50,7 @@
 /obj/item/material/get_base_value()
 	var/material/M = SSmaterials.all_materials[material_id]
 	. = M.value_per_unit * material_multiplier
+	. *= amount
```

Comment:
Actually I'm 90% sure that the amount is already considered in final value calculations.